hardware/Netatmo.cpp

```diff
@@ -488,14 +526,146 @@ uint64_t CNetatmo::convert_mac(std::string mac)
 }
 
 
+uint64_t getValueFromString(const std::string& value)
```

Comment:
Instead of this function please use **std::stoll**

Reply:
OK

Reply:
Done